.gitignore

```diff
@@ -1,2 +1,3 @@
 bower_components/
 node_modules/
+dist/
```

Comment:
Open question: Do we drop bower support and roll with npm? If we still want bower, the `dist/`directory will need to be committed after each build.